src/index.ts

```diff
@@ -2,7 +2,7 @@ export { name, version } from '../package.json';
 import GoslingSchema from '../schema/gosling.schema.json';
 import ThemeSchema from '../schema/theme.schema.json';
 
-export type { GoslingSpec, TemplateTrackDef } from './core/gosling.schema';
+export type { GoslingSpec, Encoding, TemplateTrackDef } from './core/gosling.schema';
```

Comment:
Might need to ensure to have `Encoding` in the `schema.json` file.

Reference: https://github.com/gosling-lang/gosling.js/issues/565#issuecomment-951922456